### PR TITLE
Recording table front end

### PR DESCRIPTION
**Pull request by hadicheaito1**
## Description

- Recordings tab works now for all recordings from all rooms that belong to the current user
- Can view the table of recordings
- Displays Name, Length, Users, Visibility, and Formats
## Testing Steps

- Pull code
- Create user
- Create room(s) for that user using their user_id
- Create recording(s) using any of those room_id's

Page should show all the recordings you make
## Screenshots (if appropriate):

image

### CODE REVIEW

File: app/javascript/hooks/queries/recordings/useRecordings.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useQuery } from 'react-query';
+import axios from 'axios';
+
+export default function useRecordings() {
+  return useQuery('getRecordings', () => axios.get('/api/v1/recordings.json', {
```
Comment (KH-Amir-TN) [inline app/javascript/hooks/queries/recordings/useRecordings.jsx:5]: Using the axios custom instance will further simplify this.

File: app/assets/stylesheets/application.bootstrap.scss [hunk 2]
```diff
@@ -8,3 +8,30 @@ body {
   background-color: whitesmoke;
 }
 
+// TODO: Hadi- Move this to specific file to only be accessible where needed
+.rooms-card {
```
Comment (KH-Amir-TN) [inline app/assets/stylesheets/application.bootstrap.scss:12]: This is duplicated in `rooms.scss`.

**Review by KH-Amir-TN — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.bootstrap.scss (hunk 2 above)
Comment (KH-Amir-TN) [reply app/assets/stylesheets/application.bootstrap.scss:12]: https://github.com/hadicheaito1/greenlight/blob/deb2fccb6215f26f82fcd6012babd7123da92184/app/assets/stylesheets/rooms.scss
https://github.com/hadicheaito1/greenlight/blob/deb2fccb6215f26f82fcd6012babd7123da92184/app/assets/stylesheets/application.bootstrap.scss#L5

Comment (KH-Amir-TN) [reply app/assets/stylesheets/application.bootstrap.scss:12]: You can change the className to something more generic `customer-info & rooms-card -> hoverable`

File: app/assets/stylesheets/application.bootstrap.scss [hunk 3]
```diff
@@ -24,3 +24,14 @@ body {
...
+.customer-info {
+  &:hover {
+    cursor: pointer;
+    background: rgba(26, 69, 74, 0.2)
+  }
+}
+
+th{
+  font-weight: normal;
+}
```
[the remark this answers is not in the document]
Comment (KH-Amir-TN) [reply app/assets/stylesheets/application.bootstrap.scss]: Agreed,

**Review by KH-Amir-TN — APPROVED** [review]

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**